GymBro/Views/SettingViews/MainSettingView.swift

```diff
@@ -8,13 +8,115 @@
 import SwiftUI
 
 struct MainSettingView: View {
+    
+    
+    // MARK: PROPERTY
+    
+    
+    /// The variable that control light mode or daek mode
+    @AppStorage("isDarkMode") private var isDarkMode = false
+    
+    /// The variable that control the weight unit
+    @AppStorage("isLb") private var isLb = true
+    
+    /// The variable that conrtrol the height unit
+    @AppStorage("isFt") private var isFt = true
+    
+    /// A variable that authorized  to apple watch, need furture improvement
+    @State var appleWatchIsOn: Bool = false
+    
+    /// A variable that authorized to apple health, need furture improvement
+    @State var appleHealthIsOn: Bool = false
+    
+    /// A variable taht will set timer on after each set is finish, need future improvement
+    @State var timerIsOn: Bool = false
+    
+    /// A url link to app stroe of GrmBro
+    @State var urlToAppStore =  "itms-apps://itunes.apple.com/developer/id463855590"
+    
+    // MARK: BODY
+    
     var body: some View {
-        Text("SETTING")
+        VStack {
+            List {
+                
+                /// First section is some pickerthat user can choose
+                Section("Unit") {
+                    Picker("weightunit", selection: $isLb) {
+                        Text("lb")
+                            .tag(true)
+                        Text("kg")
+                            .tag(false)
+                    }
+                    .pickerStyle(SegmentedPickerStyle())
+                    
+                    Picker("heightunit", selection: $isFt) {
+                        Text("ft")
+                            .tag(true)
+                        Text("cm")
+                            .tag(false)
+                    }
+                    .pickerStyle(SegmentedPickerStyle())
+                }
+                
+                /// Second section is osme switch
```

Comment:
what does `osme` stand for?

Reply:
That's a typo, I was trying to write "some"

Reply:
OK!